tsl/src/bgw_policy/compression_api.c

```diff
@@ -96,6 +96,21 @@ policy_compression_get_compress_after_interval(const Jsonb *config)
 	return interval;
 }
 
+Interval *
+policy_compression_get_compress_created_before_interval(const Jsonb *config)
```

Comment:
Like the other functions in this file. This convenience function has been added for the sake of completeness. It might get used later. So, all code cov warnings can be ignored for this function for now.